internal/services/servicebus/servicebus_namespace_disaster_recovery_config_data_source.go

```diff
@@ -50,6 +51,12 @@ func dataSourceServiceBusNamespaceDisasterRecoveryConfig() *pluginsdk.Resource {
 				AtLeastOneOf: []string{"namespace_id", "resource_group_name", "namespace_name"},
 			},
 
+			"auth_rule_id_of_the_alias": {
```

Comment:
I think this would be better as
```suggestion
			"alias_authorization_rule_id": {
```